jscripts/bbcodes_sceditor.js

```diff
@@ -474,7 +474,7 @@ $(function ($) {
 				'</div>' +
 				'<div>' +
 				'<label for="link">' + editor._('Video URL:') + '</label> ' +
-				'<input type="text" id="videourl" value="http://" />' +
+				'<input type="text" id="videourl" placeholder="http://" />' +
```

Comment:
As mentioned [here](https://github.com/mybb/mybb/issues/3721#issuecomment-502755183) the `placeholder` attribute is for instructions, and `http://` doesn't fit the bill. I would suggest creating a new language entry with something like `paste URL here` instead of using the placeholder attribute for an unintended purpose.

Reply:
Well sceditor already use it for normal links and images, if we change it here we must change it also for the other fields.

Reply:
The description of the `placeholder` attribute:

>Specifies a short hint that describes the expected value of the input field

No matter what is going on elsewhere in the code, we do not want to introduce bad design into our software.

Reply:
I think this falls into the definition of _hint_. If no one decides to push a different approach I think this could be merged.

Reply:
One issue is with https, it is missing in the "hint"

Reply:
But `http` is a _hint_ that `https` could be used. I mean, if somebody wants to create a new PR, go ahead, otherwise I see no reason to reject this contribution.